### PR TITLE
Setup notification service in mobile app

### PR DESCRIPTION
**Pull request by adube**
This PR introduces the notification service in the mobile app.

## Todo

 * [x] review

## Live demo

Try to login with bad credentials or try to refuse to share your position when using the geolocation tool.

 * https://adube.github.io/ngeo/notification-in-mobile-app/examples/contribs/gmf/apps/mobile

### CODE REVIEW

**Comment by adube** [comment]
Ready for review.

**Comment by sbrunner** [comment]
Looks good for me

File: contribs/gmf/less/mobile.less [hunk 1]
```diff
@@ -129,6 +129,15 @@ button[ngeo-mobile-geolocation] {
   }
 }
 
+.ngeo-notification {
+  left: 50%;
+  margin: 0 0 0 -100px;
```
Comment (sbrunner) [inline contribs/gmf/less/mobile.less]: Same as in desktop we should use rem unit...

**Comment by adube** [comment]
Fixed.  Live demo not updated.

**Pull request merged**